tensorboard/components/vz_example_viewer/vz-example-viewer.ts

```diff
@@ -371,21 +372,19 @@ Polymer({
   },
 
   haveSaliency: function() {
+    // Saliency-coloring waits until the display elements have been updated
+    // to avoid coloring divs that are then re-ordered/re-used/re-named by
+    // the dom-repeat of feature divs.
+    this.debounce('haveSaliency', () => this.haveSaliencyImpl(),
+                  ATTRIBUTION_TIMER_DELAY_MS);
```

Comment:
Curious, what is the reasoning behind the number `200`?

Reply:
Just manual testing for an experience where we see correct rendering without a significant delay.

Reply:
I am a little bit worried about such magic number :(
Does waiting a requestAnimationFrame work?

Reply:
Perhaps I'm being too conservative. requestAnimationFrame seems to work in all cases I've tried. Will use requestAnimationFrame and keep an eye out for issues.